openstates/md/bills.py

```diff
@@ -218,7 +218,8 @@ def parse_vote_pdf(self, vote_url, bill):
 
         passed = vote_counts[0] > vote_counts[1]
         motion = re.split(r'\s{2,}', lines[page_index-3].strip())[0]
-        motion_keywords = ['favorable', 'reading', 'amendment', 'motion']
+        motion_keywords = ['favorable', 'reading', 'amendment', 'motion',
+                           'shall the bill be introduced?']
```

Comment:
Since these are keywords, rather than full motion names, let's go with `introduced` instead of the entire motion.

Reply:
@mileswwatkins I also thought to add ```introduced``` but this word seems to be used a lot around in the pdf, to which I thought let's add the whole text.

Reply:
Although I can still break it to ```bill be introduced``` which seems should work I think.

Reply:
Fair point, for sure. I spot-checked a few additional bill PDFs, and `introduced` seems to be safe to use.